cmd/installer/root_test.go

```diff
@@ -258,12 +259,14 @@ var (
 	pack123VersionNotLatest   = filepath.Join(testDir, "TheVendor.LocalPackWithVersionNotLatest.1.2.3.pack")
 
 	// Public packs
-	publicLocalPack010  = filepath.Join(testDir, "0.1.0", "TheVendor.PublicLocalPack.0.1.0.pack")
-	publicLocalPack011  = filepath.Join(testDir, "0.1.1", "TheVendor.PublicLocalPack.0.1.1.pack")
-	publicLocalPack122  = filepath.Join(testDir, "1.2.2", "TheVendor.PublicLocalPack.1.2.2.pack")
-	publicLocalPack123  = filepath.Join(testDir, "1.2.3", "TheVendor.PublicLocalPack.1.2.3.pack")
-	publicLocalPack124  = filepath.Join(testDir, "1.2.4", "TheVendor.PublicLocalPack.1.2.4.pack")
-	publicRemotePack123 = filepath.Join(testDir, "1.2.3", publicRemotePack123PackID+".pack")
+	publicLocalPack010     = filepath.Join(testDir, "0.1.0", "TheVendor.PublicLocalPack.0.1.0.pack")
+	publicLocalPack011     = filepath.Join(testDir, "0.1.1", "TheVendor.PublicLocalPack.0.1.1.pack")
+	publicLocalPack122     = filepath.Join(testDir, "1.2.2", "TheVendor.PublicLocalPack.1.2.2.pack")
+	publicLocalPack123     = filepath.Join(testDir, "1.2.3", "TheVendor.PublicLocalPack.1.2.3.pack")
+	publicLocalPack124     = filepath.Join(testDir, "1.2.4", "TheVendor.PublicLocalPack.1.2.4.pack")
+	publicLocalPack123Pdsc = filepath.Join(testDir, "1.2.3", "TheVendor.PublicLocalPack.pdsc")
+	publicLocalPack124Pdsc = filepath.Join(testDir, "1.2.4", "TheVendor.PublicLocalPack.pdsc")
+	publicRemotePack123    = filepath.Join(testDir, "1.2.3", publicRemotePack123PackID+".pack")
```

Comment:
What's happening here? Just a realignment due to gofmt correct?

Reply:
Exactly!